content/articles/image-watermarking-with-opencv-python/index.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: engineering-education
+status: draft
+published: false
+url: /image-watermarking-with-opencv-python/
+title: Multiple Image Watermarking Using OpenCV with Python
+description: The objective of this tutorial is to help the reader understand the concept of how one can automate the process of image watermarking on multiple images with less effort and time.
+author: atonya-dennis
+date: 2021-09-22T00:00:00-10:16
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/image-watermarking-with-opencv-python/hero.jpg
+    alt: OpenCV Example image
+---
+
+Watermarks play an important part in businesses and online content credit making. They provide the author of digital artifacts rights or credits, preventing the content from being stolen or duplicated without the owner's permission. They could be as a distinctive and one-of-a-kind logo, signature, or stamp. Making these watermarks using Photoshop or manually requires more effort and time. We made this easier than ever using OpenCV with Python to automate watermarking multiple images.     
```

Comment:
This introduction statement is not purely original. I easily found it on this [blog post](https://www.analyticsvidhya.com/blog/2021/08/how-to-add-watermark-on-images-using-opencv-in-python/). Please not that we only publish original content. Re-write this section accordingly.